Chandra/test_Time.py

```diff
@@ -186,6 +186,20 @@ def test_leapsec_2015(self):
         t2 = DateTime('2015-07-01 00:00:00').secs
         self.assertAlmostEqual(t2 - t1, 1.0)
 
+    def test_leapsec_2016(self):
+        """
+        Test that there are 4 clock ticks where one usually expects 3 (PR #15).
```

Comment:
Of the three setups, is it just the last one that you'd consider `initializing within the leap sec`?